src/Components/Gene/GeneRelatedLinks.tsx

```diff
@@ -0,0 +1,91 @@
+import { Box } from "@artsy/palette"
+import {
+  GeneRelatedLinksQuery,
+  GeneRelatedLinksQueryResponse,
+} from "__generated__/GeneRelatedLinksQuery.graphql"
+import { useSystemContext } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import { SystemQueryRenderer as QueryRenderer } from "Artsy/Relay/SystemQueryRenderer"
+import React from "react"
+import { graphql } from "react-relay"
+
+/**
+ * Used on the Gene page to show related artists and categories.
+ */
+export const GeneRelatedLinks: React.FC<GeneRelatedLinksQueryResponse> = ({
+  gene,
+}) => {
+  return (
+    <>
+      <Box className="related-genes related-links bisected-header-cell-section is-fade-in">
+        <h2>Related Categories</h2>
+        <div className="related-genes-links">
+          {gene.similar.edges.map(({ node: similarGene }, index) => {
```

Comment:
Is it possible that `gene.similar` would return null and we should be more defensive about the presence of `edges`?

Reply:
Certainly possible. I'll wrap it in a check.